contracts/ServiceNodeRewards.sol

```diff
@@ -423,7 +423,7 @@ contract ServiceNodeRewards is Initializable, Ownable2StepUpgradeable, PausableU
         }
 
         updateBLSNonSignerThreshold();
-        emit NewServiceNodeV2(allocID, operator, blsPubkey, serviceNodeParams, contributors);
```

Comment:
Whats the reasoning behind changing this to `tx.origin`? operator was previously `contributors[0].staker.addr` and if the operator isnt the one calling `addBLSPublicKey` (like if it automatically fills) then tx.origin will be the last contributor

Reply:
The event calls that parameter "initiator" which isn't true if a contributor calls the function (via filling up the node). Operator also wasn't particularly useful as we know the operator from the caller (if solo) or from the multi-contrib contract deploy. 
Would need to either:
- Change to `tx.origin`
- Change name of event param to `operator`

went with `tx.origin` to be consistent and its more useful. also changed all calls to include the initiator for consistency (and it may be useful)